src/main/java/nl/tudelft/lifetiles/graph/controller/GraphController.java

```diff
@@ -22,8 +30,28 @@
     private ScrollPane wrapper;
 
     @Override
-    public void initialize(final URL location, final ResourceBundle resources) {
-        // Initialize.
+    public final void initialize(final URL location,
+            final ResourceBundle resources) {
+        Tile model = new Tile(loadGraphModel());
+        TileView view = new TileView();
+        TileController controller = new TileController(view, model);
+
+        Group root = controller.drawGraph();
+        wrapper.setContent(root);
+    }
+
+    /**
+     * Loads a graph by filename.
+     *
+     * @return parsed graph by filename.
+     */
+    private Graph<SequenceSegment> loadGraphModel() {
```

Comment:
I assume this is an obsolete method? If that's the case you should mark it with `@Deprecated`.